.github/workflows/comment-body.md

```diff
@@ -21,7 +22,7 @@
       "github_trial_issue_link": "https://github.com/holdex/trial/issues/your_issue_number"
     }
     ```
-3. Create a Pull Request (PR), [attach it to this issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#about-linked-issues-and-pull-requests) and wait for the review.
+3. Create a Pull Request (PR), [attach it to this issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#about-linked-issues-and-pull-requests) and wait for the review. Double check to make sure your PR matches perfectly with our [Guidelines](https://github.com/holdex/developers/blob/main/.github/CONTRIBUTING.md). Otherwise, we won't consider your submission.
```

Comment:
@tiffmyaaw 

Revert this line


There is no need to remind people, especially what we have written exactly 14 lines above. We are not parrots who repeat things; thus, we are looking for smart people. Let's use it as a filter.